Lib/test/test_threading.py

```diff
@@ -2103,55 +2103,50 @@ def test__all__(self):
         support.check__all__(self, threading, ('threading', '_thread'),
                              extra=extra, not_exported=not_exported)
 
+    @unittest.skipUnless(hasattr(_thread, 'set_name'), "missing _thread.set_name")
```

Comment:
There is also `test.support.get_attribute()` which converts AttributeError to SkipTest. What to use is a matter of personal preferences.

Reply:
Oh.. thank you for making me discover this one!

Reply:
I'm more used to `unittest.skipUnless()` :-)